CPDShell/main.py

```diff
@@ -12,56 +10,50 @@
 from CPDShell.generator.saver import DatasetSaver
 from CPDShell.shell import CPDShell
 
-with tempfile.TemporaryDirectory() as tempdir_graph:
-    path_string = "tests/test_CPDShell/test_configs/test_config_exp.yml"
-    distributions_name = "exp"
+path_string = "tests/test_CPDShell/test_configs/test_config_exp.yml"
+distributions_name = "exp"
 
-    saver = DatasetSaver(Path(), True)
-    generated = ScipyDatasetGenerator().generate_datasets(Path(path_string), saver)
-    data, actual_change_points = generated[distributions_name]
+saver = DatasetSaver(Path(), True)
+generated = ScipyDatasetGenerator().generate_datasets(Path(path_string), saver)
+data, actual_change_points = generated[distributions_name]
 
-    graph_cpd = CPDShell(data)
-    graph_cpd.scrubber.window_length = 150
-    graph_cpd.scrubber.movement_k = 2.0 / 3.0
+graph_cpd = CPDShell(data)
+graph_cpd.scrubber.window_length = 150
+graph_cpd.scrubber.movement_k = 2.0 / 3.0
 
-    print("Actual change points:", actual_change_points)
+print("Actual change points:", actual_change_points)
 
-    res_graph = graph_cpd.run_cpd()
-    res_graph.visualize(True)
-    print("Graph algorithm")
-    print(res_graph)
+res_graph = graph_cpd.run_cpd()
+res_graph.visualize(True)
```

Comment:
It seems that the .visualize function already has the output Actual points change.

Reply:
> It seems that the .visualize function already has the output Actual points change.

I printed them in console so we could see it in pair with our results. Also I cannot see actual change points on the plot, only the localized ones.

![image](https://github.com/user-attachments/assets/7c8eeb23-023a-4520-a0e8-8241dbcba76d)

Reply:
Changed naming